<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Copyright 2010, Martin Koller</author>
 <class>ClosedTickets</class>
 <widget class="QDialog" name="ClosedTickets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Abgeschlossene Bons</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Account</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date / Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Table</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Paid with</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sum</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">sortDate</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="print">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Print</string>
       </property>
       <property name="icon">
        <iconset resource="LillePOS.qrc">
         <normaloff>:/icons/print.png</normaloff>:/icons/print.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Looking statements:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchInvoice">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showPrivate">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Zeige Private 
Bons</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showAllDays">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Show all days</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="change">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Change</string>
       </property>
       <property name="icon">
        <iconset resource="LillePOS.qrc">
         <normaloff>:/icons/edit.png</normaloff>:/icons/edit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="LillePOS.qrc">
         <normaloff>:/icons/ok.png</normaloff>:/icons/ok.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="LillePOS.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ClosedTickets</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>687</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>672</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
